assignments/arrays.js

```diff
@@ -90,7 +100,13 @@ console.log();
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi =[];
-console.log();
+
+for (let i = 0;i<inventory.length;i++) {
+    if (inventory[i].car_make === "BMW" || "Audi") {
+        BMWAndAudi.push (inventory[i])
```

Comment:
This `if` statement isn't complete. Let me know if you need help figuring it out.